lib/identity/hostdata/version.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 module Identity
   module Hostdata
-    VERSION = '3.1.2'
+    VERSION = '3.1.3'
```

Comment:
this is a medium size change since it's using a different HTTP endpoint, I'd consider a bigger bump

```suggestion
    VERSION = '3.2.0'
```